docs/topics/control-flow.md

```diff
@@ -6,30 +6,52 @@ In Kotlin, `if` is an expression: it returns a value.
 Therefore, there is no ternary operator (`condition ? then : else`) because ordinary `if` works fine in this role.
 
 ```kotlin
-var max = a 
-if (a < b) max = b
+val tax: Int
+val value = 50
+
+if (value < 100) tax = 10
 
 // With else 
-var max: Int
-if (a > b) {
-    max = a
+val tax: Int
+val value = 50
+
+if (value < 10) {
+    tax = 1
 } else {
-    max = b
+    tax = 10
 }
  
 // As expression 
-val max = if (a > b) a else b
```

Comment:
I would propose returning everything as it was and just adding a couple of lines like this:

> // You can also use `else if` branches in expressions:
> val maxOrLimit = if (maxLimit > a) maxLimit else if (a > b) a else b

We can discuss our approach to examples on our next sync.

Reply:
@PetrakovichVictoria, I've made changes as we discussed over Slack. Is this good to go?